src/test/java/api/loans/CheckInByBarcodeTests.java

```diff
@@ -919,6 +924,78 @@ public void noOverdueFineShouldBeChargedForOverdueFinePolicyWithNoOverdueFine()
     assertThat("Fee/fine action record shouldn't be created", createdFeeFineActions, empty());
   }
 
+  @Test
+  public void overdueFineShouldBeChargedWhenItemIsOverdueForSevenHours() {
```

Comment:
I think method name is misleading

Reply:
@alexanderkurash Please could you expand upon your feedback. Why do you think it is misleading? What might you suggest as an alternative?

Reply:
@marcjohnson-kint I don't think it reflects the purpose of the test. It looks like we're trying to check that hourly policies are working correctly, not that we charge a fine when item is exactly 7hrs overdue. I would probably go with something like overdueFineCalculatedCorrectlyWhenHourlyFeeFinePolicyIsApplied

Reply:
renamed